web-ui/src/main/resources/catalog/templates/admin/harvest/type/webdav.html

```diff
@@ -98,6 +98,22 @@
     <fieldset>
       <legend data-translate="">filteringAndProcessing</legend>
 
+      <div id="gn-harvest-settings-gn-advanced-xsl-row">
+        <label
+          id="gn-harvest-settings-gn-advanced-xsl-label"
+          class="control-label"
+          data-translate=""
+          >geonetwork-xslfilter</label
+        >
+        <input
+          id="gn-harvest-settings-gn-advanced-xsl-input"
+          type="text"
+          class="form-control"
+          data-ng-model="harvesterSelected.site.xslfilter"
+        />
```

Comment:
```suggestion
        <div
          id="gn-harvest-settings-gn-advanced-xsl-input"
          data-gn-import-xsl="harvesterSelected.site.xslfilter"
          data-mode="btn-group"
        ></div>
```
to use the selector
![image](https://github.com/user-attachments/assets/456450c1-75dd-40bf-9ff3-205644a99c6d)